### DISCUSSION TITLE
Welcome to HotelOnTouch Discussions! Actively looking for new contributors and Mentors

### DISCUSSION THREAD

**Discussion by ayan-biswas0412** [ayan-biswas0412/gssoc2021-HotelOnTouch#6] [General]
## 👋 Welcome to HotelOnTouch Project !
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## For New Participants

 * Please see this if you are a new participant [Participant Onboarding](https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/wiki/Contributor-Onboarding)

Comment below  related for any new ideas, issues of any bugs or feature or anything

**Comment by ayan-biswas0412** [comment]
See our introductory video | Contribute to our project 🌎🌎

https://user-images.githubusercontent.com/52851184/107491257-ea0bf380-6bb0-11eb-8dd9-e895d56fcc3c.mp4

**Reply by JanviMahajan14** [reply]
Hello @ayan-biswas0412 . I am new to this project. Could you please make a slack or discord channel where everyone can discuss about the project. Thanks!

**Reply by vinit-churi** [reply]
yes please that would be helpful

**Reply by ayan-biswas0412** [reply]
@JanviMahajan14 yah I also thought about that but I am not sure about if it is officially acceptable for GSSoc 2021 to make that once I get the confirmation I will do that

**Comment by ashirbad29** [comment]
Hello, I am ashirbad behera
I am a CS student and passionate about opensource 
I am familiar with react and nodeJS

**Comment by tanay13** [comment]
Hello I am Tanay Raj, I am a CS undergraduate and I am interested in backend development using nodeJS

**Comment by vinit-churi** [comment]
Hello I am Vinit Churi, currently a CS undergraduate, I am familiar with react and nodeJS

**Comment by JanviMahajan14** [comment]
Hello Everyone! I am happy to share with you all that I will be Mentoring this project as a part of Girlscript Summer of Code'21. I am enthusiastic about web development and Open Source. I am looking forward to help the enthusiastic participants and contribute in the project :) Thanks!

**Reply by vinit-churi** [reply]
If we have any doubt while programming a feature or solving a bug, 
can we tag you in the that issue thread?

**Reply by ayan-biswas0412** [reply]
you can always tag mentors and admin if you stuck at any point

**Reply by ayan-biswas0412** [reply]
> Hello Everyone! I am happy to share with you all that I will be Mentoring this project as a part of Girlscript Summer of Code'21. I am enthusiastic about web development and Open Source. I am looking forward to help the enthusiastic participants and contribute in the project :) Thanks!

hi @JanviMahajan14 I am the project admin of this project can you forward the email (if you got any from GSSoc 2021) to ayanbiswas184@gmail.com because I haven't got your info till now from GSSoc end so it will be easier for me to communicate further

**Reply by JanviMahajan14** [reply]
@ayan-biswas0412 Sure! I have forwarded the mail to you.

**Reply by ayan-biswas0412** [reply]
> @ayan-biswas0412 Sure! I have forwarded the mail to you.

yah got it thanks

**Comment by khareyash05** [comment]
Hi! Myself Yash Khare, currently a CS undergraduate ... I have  a knowledge of ReactJS.,Node JS, and MongoDB

**Reply by ayan-biswas0412** [reply]
Welcome to the project 🥰

**Comment by Anjali041** [comment]
Hello Everyone! I am Anjali, currently an undergraduate. I am familiar with the Mern stack.

**Reply by JanviMahajan14** [reply]
Hi @Anjali041 Welcome to the project. Feel free to browse the issues in the project and let us know if you need any kind of help :)

**Comment by abhishek7997** [comment]
Hello everyone, I am Abhishek, currently doing B.Tech in Computer Science. I am familiar with C++, python, flutter, react.js, node. and mongodb.
I also do a bit of graphic design.
Insta page: https://www.instagram.com/designxd_ai

**Comment by ayan-biswas0412** [comment]
please have a look at https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/discussions/92

**Comment by Prajwal-ctrl** [comment]
Hello, I am Prajwal Sontakke
I am Sophomore and passionate about learning and contributing in opensource 
I am familiar with MERN STACK

**Reply by ayan-biswas0412** [reply]
Welcome @Prajwal-ctrl 🥰

**Reply by ayan-biswas0412** [reply]
@Prajwal-ctrl you can start from here #92

**Comment by nehathakur27** [comment]
Hi! I'm Neha Thakur and currently I'm pursuing my master's focused in computer applications. I'm familiar with ReactJS , React Native, Java,Python etc.

**Reply by ayan-biswas0412** [reply]
Hi @nehathakur27 welcome to the community you can refer this #92 for some new contributions

**Comment by singhayushh** [comment]
Hola!
I'm Ayush Singh, 2nd year undergrad student in ECE. I have worked with in backend development in nodejs as well as golang. Looking forward to make a lot of contributions to this project!

**Comment by rohithmsr** [comment]
Hello! I am Rohith M S R, currently doing B.Tech in IT.
I am familiar with MERN stack and Flutter.

**Reply by ayan-biswas0412** [reply]
welcome @rohithmsr to the community , have a look at the issues ,Thank you

**Comment by ayan-biswas0412** [comment]
guys please star ⭐️ the repo https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch I will be more than happy if you do that

**Comment by ayan-biswas0412** [comment]
Can you please star the repo please https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch 🥰 🥰If you guys have enjoyed working please star the repo

**Comment by AshuKV** [comment]
Hello, Myself Ashutosh Verma
I am familiar with MERN stack. Looking forward to contributing to this project!!!

**Reply by ayan-biswas0412** [reply]
welcome to the community make sure you have joined project discord channel

**Comment by ayan-biswas0412** [comment]
If I have helped you guys to learn something new please nominate me as a Github Star , This is just a simple procedure Nominate me here : https://stars.github.com/nominate/

**Comment by satyampandeygit** [comment]
Hello everyone, myself Satyam Pandey. I am learning frontend technologies and will try to contribute to this project. Thank you for your work @ayan-biswas0412

**Comment by hsnice16** [comment]
Hello everyone, I am Himanshu Singh, currently pursuing CS honors.

**Reply by ayan-biswas0412** [reply]
welcome to the community @hsnice16

**Comment by ayan-biswas0412** [comment]
I am looking for a new maintainer for this project, interested candidates may mail their cv at ayan.biswas@judh.in
with their cv (please only provide your development background)

Mail Subject: Application for Maintainers | HotelOnTouch

**Comment by AB-san** [comment]
Hello everyone I am Ansuman Behera , a GSSOC21 participant , currently pursuing BTECH CSE , I am familiar with front end web dev part . feel free get in touch :)

**Comment by mishrajiharsh219** [comment]
Hello, I am Harsh Mishra
I am a CS student and passionate about opensource

**Comment by kumarishalini6** [comment]
Hello, I am shalini , open source passionate

**Comment by ogofe** [comment]
Hello, I'm **Joel Tanko**.
I'm a Nigerian developer, I'm passionate about technology and contributing to the open source community.
I hope to make some useful contributions to this project.

## My Tech Stack and Languages
- Python3
- Typescript and Javascript
- MongoDB
- PostgreSQL
- React and React Native
- Django
- GraphQL and REST
-  AWS

I've mostly worked on hobby and freelance projects, but I have 2 years of professional working experience as a Full-stack developer, I've created an - e-commerce marketplace, - a free-time checker, and now, I'm working on a logistics and delivery software for the African ecosystem.

***Great to be here and I hope to work on something real soon. Cheers.***
